external/test-runtime/XUnit.Runtime.depproj

```diff
@@ -140,7 +140,10 @@
           AfterTargets="RestorePackages"
           Condition="'$(TargetGroup)' == 'uapaot'">
     <PropertyGroup>
-      <TestILCToolsPackageName>TestILCNugetPackageForCoreFX</TestILCToolsPackageName>
+      <TestILCToolsPackageName>TestILC</TestILCToolsPackageName>
+      <TestILCToolsPackageName Condition="'$(ArchGroup)' == 'x64'">$(TestILCToolsPackageName).amd64ret</TestILCToolsPackageName>
+      <TestILCToolsPackageName Condition="'$(ArchGroup)' == 'x86'">$(TestILCToolsPackageName).x86ret</TestILCToolsPackageName>
+      <TestILCToolsPackageName Condition="'$(ArchGroup)' == 'arm'">$(TestILCToolsPackageName).armret</TestILCToolsPackageName>
```

Comment:
I tested that, if it happens then we will only throw a warning since we couldn't find where to copy ILC from, but this shouldn't stop you from building the product. If you actually want to run tests and didn't copy the TestILC, then we will throw an error with a friendly message saying that we couldn't find TestILC.